src/Agent.Listener/Configuration/NativeWindowsServiceHelper.cs

```diff
@@ -760,6 +759,19 @@ private byte[] GetSidBinaryFromWindows(string domain, string user)
             }
         }
 
+        private NTAccount GetServiceAccount(WellKnownSidType sidType)
+        {
+            SecurityIdentifier sid = new SecurityIdentifier(sidType, domainSid: null);
+            NTAccount account = sid.Translate(typeof(NTAccount)) as NTAccount;
+
+            if (account == null)
+            {
+                throw new InvalidOperationException(StringUtil.Loc("NetworkServiceNotFound"));
```

Comment:
this loc string is incorrect when attempting to resolve local system. i would just delete this shared method and copy the three lines into the two places that call this method - and add a new string for case when resolving local system.